app/containers/Sessions/containers/QuestionListGroup/QuestionGroupSettingsModal/index.tsx

```diff
@@ -146,6 +146,27 @@ const QuestionGroupSettingsModal = ({
             px={12}
           />
         </Row>
+        {questionGroup.type === GroupType.INITIAL && (
+          <>
+            <Row justify="between" align="center" mb={8}>
+              <H3>Licza powtórzeń</H3>
```

Comment:
I think we can move it to the `QuestionGroupSettingsModal/messages.js` file and the translation to the `app/translations/en.js`